### PR TITLE
ovmm agent for evalai

### PR DESCRIPTION
**Pull request by yvsriram**
## Motivation and Context
1. Separate out agent from evaluation class

## Types of changes

- [ ] Docs change / refactoring / dependency upgrade
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

# Checklist:

- [ ] I have made changes to existing documentation where needed.
- [ ] I have added tests that show that the PR is functional.
- [ ] I ran `tests/hw_manual_test.py` on hardware and observed that robot behavior looks normal.

### CODE REVIEW

File: projects/habitat_ovmm/eval_dataset.py [hunk 1]
```diff
@@ -49,31 +50,49 @@ def create_ovmm_env_fn(config):
...
+    episode_ids_range = config.habitat.dataset.episode_indices_range
+    if episode_ids_range is not None:
+        config.EXP_NAME = os.path.join(
+            config.EXP_NAME, f"{episode_ids_range[0]}_{episode_ids_range[1]}"
+        )
+    OmegaConf.set_readonly(config, True)
+    return config
+
+
+class VectorizedEvaluator(PPOTrainer):
```
Comment (rpartsey) [inline projects/habitat_ovmm/eval_dataset.py:92]: Can we move VectorizedEvaluator class to the separate file (for example, `vectorized_evaluator.py`) and then import it in the `eval_dataset.py`?

After we move VectorizedEvaluator to the separate file can we also rename `eval_dataset.py` to `eval_ovmm_agent.py` or better `eval_baseline_ovmm_agent.py`? 

Then we can define Random agent in `eval_random_ovmm_agent.py` (similar to how it is done in [habitat-challenge](https://github.com/facebookresearch/habitat-challenge/blob/main/agents/agent.py)).

**Review by rpartsey — APPROVED** [review]
Overall looks good, lets refactor the code a bit by moving classes into separate files like I suggested in the comment.

File: projects/habitat_ovmm/eval_dataset.py (hunk 1 above)
Comment (yvsriram) [reply projects/habitat_ovmm/eval_dataset.py]: Updated. Please check.

Comment (rpartsey) [reply projects/habitat_ovmm/eval_dataset.py]: @yvsriram LGTM, evaluation code runs on my machine. Thank you!

Comment (yvsriram) [reply projects/habitat_ovmm/eval_dataset.py]: Thanks for testing.

File: projects/habitat_ovmm/README.md [hunk 2]
```diff
@@ -103,27 +103,31 @@ python -u -m habitat_baselines.run \
 
 ## Running evaluations
 
-```
-cd /path/to/home-robot
-
 
-# Evaluation on complete episode dataset with GT semantics
-python projects/habitat_ovmm/eval_dataset.py
+### Evaluate with GT semantics
```
Comment (cpaxton) [inline projects/habitat_ovmm/README.md:107]: Write out ground truth

**Review by cpaxton — APPROVED** [review]
lgtm, minor comments

**Pull request merged**